test/e2e/README.md

```diff
@@ -192,6 +192,41 @@ You can load the new image into the kind cluster using the command below:
 kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:latest --name <kind_cluster_name>
 ```
 
+## Running the e2e tests on an existing K8s cluster
+
+If you already have a K8s cluster on which Antrea has been deployed, the following steps should be done for running the e2e tests.
+
+First, you should config the ssh information for each Node in the cluster. Here is an example:
+
+```bash
+Host <Control-Plane-Node>
+    HostName <Control-Plane-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+Host <Worker-Node>
+    HostName <Worker-Node-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+```
+
```

Comment:
some of the lines below are too long; we typically wrap around 80 to 100 characters